File: intersection.js
var firstArray = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
var secondArray = [2, 4, 6, 8, 10];
var result = [];
var intersection = function (firstArray, secondArray) {
	for (var i = 0; i < firstArray.length; i++) {
		for (var j = 0; j < secondArray.length; j++) {
			if (firstArray[i] === secondArray[j]) {
				result.push(firstArray[i]);
				break;
			}
		}
	}
	return result;
};
result = intersection(firstArray, secondArray);
console.log(result);

File: intersection.py
firstArray = [1, 2, 3, 4, 5]
secondArray = [1, 4, 8, 3, 6]

def intersection(firstArray, secondArray):
    result = []

    for i in range(len(firstArray)):
        for j in range(len(secondArray)):
            if firstArray[i] == secondArray[j]:
                result.append(firstArray[i])
    return result
    

    


result = intersection(firstArray, secondArray)
print(result)

